interpreter/src/test/scala/pl/touk/nussknacker/engine/spel/TyperSpec.scala

```diff
@@ -86,11 +87,16 @@ class TyperSpec extends AnyFunSuite with Matchers with ValidatedValuesDetailedMe
       TypedObjectWithValue(Typed.typedClass[String], "nestedVal1")
   }
 
+  test("dynamic property access on records returns error when disabled") {
```

Comment:
Let's add a test in `SpelExpressionSpec` as well. It is more higher level, giving a quick grasp, what is allowed in expressions

Reply:
Done